engine/src/cmd/script/mission.h

```diff
@@ -476,7 +476,7 @@ class Mission
     static double gametime;
     string mission_name;
     void terminateMission();
-    Unit  *call_unit_launch( class CreateFlightgroup*fg, int type /*clsptr type*/, const std::string&destinations );
+    Unit  *call_unit_launch( class CreateFlightgroup*fg, int type /*_UnitType type*/, const std::string&destinations );
```

Comment:
this looks like the type of the parameter should be changed from `int` to `_UnitType`

Reply:
As noted above, I'm trying not to touch every kludge I see. Yes, it should be _UnitType and once we switch to enum class, this code right here would fail and we'd need to investigate and it would more resilient after we refactor it.
But this PR is big enough without more code in it and we have #359 for it.